app/controllers/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
+
+  def new
+  end
+
+  def create
+    new_user = User.create(user_params)
+    flash[:success] = "Hello #{new_user.first_name} #{new_user.last_name}, thank you for creating an account!"
```

Comment:
Need to move this to another line, so it does not go over character limit

Reply:
In my opinion, a flash message going over the character limit for the method line isn't nearly as bad as an actual method that goes on and on and crosses the line. So I would say it is totally fine, but other people may have better suggestions!